### PR TITLE
chore(README): update readme format

### PR DESCRIPTION
**Pull request by sean-perkins**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

[Preview Changes](https://github.com/ionic-team/ionic-framework/blob/f8ed3f63307f6dc0688d5e0d1e776d311c2a1b3f/README.md)

Introduce branding and content changes to the README.

- Adds migration guides section 
- Adds download metrics for independent packages
- Adds community links (discord, forums, twitter)

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

Need to update the link for v6 migration to latest once `/next` docs are merged into main (FW-416).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,63 @@
...
+</p>
+
+<h2 align="center">
+  <a href="https://ionicframework.com/getting-started/">Quickstart</a>
+  <span> · </span>
+  <a href="https://ionicframework.com/docs/">
+    Documentation
+  </a>
+  <span> · </span>
+  <a href="https://ionicframework.com/docs/contributing/how-to-contribute">Contribute</a>
```
Comment (amandaesmith3) [inline README.md:41]: It looks like this page is missing some content from the contributing doc the original readme linked to: https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md Maybe we should go with that for now?

File: README.md [hunk 2]
```diff
@@ -26,6 +67,18 @@ Start a new project by following our quick [Getting Started guide](https://ionic
 We would love to hear from you! If you have any feedback or run into issues using our framework, please file
 an [issue](https://github.com/ionic-team/ionic/issues/new) on this repository.
 
+### Migration Guides
+
+Already have an Ionic app? These guides will help you migrate to the latest versions.
+
+* [Migrate from v5 to v6](https://ionicframework.com/docs/next/reference/migration#migrating*from*ionic-5x-to-ionic-6x)
+* [Migrate from v4 to v5](https://ionicframework.com/docs/reference/migration#migrating*from*ionic-4x-to-ionic-5x)
+* [Migrate from v3 to v4](https://ionicframework.com/docs/reference/migration#migrating*from*ionic-30-to-ionic-40)
```
Comment (amandaesmith3) [inline README.md:76]:
```suggestion
* [Migrate from v5 to v6](https://ionicframework.com/docs/next/reference/migration#migrating-from-ionic-5x-to-ionic-6x)
* [Migrate from v4 to v5](https://ionicframework.com/docs/reference/migration#migrating-from-ionic-4x-to-ionic-5x)
* [Migrate from v3 to v4](https://ionicframework.com/docs/reference/migration#migrating-from-ionic-30-to-ionic-40)
```

**Review by amandaesmith3 — COMMENTED** [review]
This looks great! Just noticed a couple minor things. I love the new branding at the top.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,63 @@
...
-build top quality native and progressive web apps with web technologies.
-
-Ionic Framework is based on [Web Components](https://www.webcomponents.org/introduction) and comes with many significant performance, usability, and feature improvements over the past versions.
-
-## Looking for the Ionic Framework v6 beta?
-
-[Click here to get started!](https://github.com/ionic-team/ionic-framework/blob/next/BETA.md)
+<p align="center">
+  <a href="#">
+    <img alt="Ionic" src="https://ionic.io/_next/image?url=https%3A%2F%2Fimages.prismic.io%2Fionicframeworkcom%2F66cfdbef-e59d-463a-8e24-12cb233e9d97_ionic%2Blogo%2Bblue.png&w=128&q=75" width="60" />
```
Comment (liamdebeasi) [inline README.md:3]: Can we commit this logo to the repo instead of loading it from a separate website?

**Review by liamdebeasi — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (sean-perkins) [reply README.md:3]: Sure thing 👍  I was unsure if we wanted the assets to be managed by the web team, but easy to remedy if they need changes in the future & hosting within Github is likely more reliable.

**Review by amandaesmith3 — APPROVED** [review]
Good to go once the logo is updated 👍

**Review by liamdebeasi — APPROVED** [review]
Good to go once logo is updated as well. Great work!

**Pull request merged**